### PR TITLE
[ES] Clean up `Modding`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Modding/es.md [hunk 1]
```diff
@@ -2,41 +2,41 @@
 
 *Para otros usos, véase [Mod (desambiguación)](/wiki/Disambiguation/Mod).*
 
-**Modding** es el acto de proveer comentarios al [beatmap](/wiki/Beatmap) de otro usuario. Un usuario que está modeando es considerado como un *modder*, y sus posts colectivos en una discusión de un beatmap se denomina como un *mod*.
+El **modding** es el acto de hacer comentarios sobre el [beatmap](/wiki/Beatmap) de otro usuario. Un usuario que modea es conocido como un *modder*, y sus mensajes colectivos en la discusión de un beatmap son conocidos como *mods*.
```
Comment (agatemosu) [inline wiki/Modding/es.md:5]:
```suggestion
**Modding** es el acto de dar comentarios sobre el [beatmap](/wiki/Beatmap) de otro usuario. Un usuario que modea es conocido como un *modder*, y sus mensajes colectivos en la discusión de un beatmap son conocidos como *mods*.
```

File: wiki/Modding/es.md [hunk 2]
```diff
@@ -2,41 +2,41 @@
...
 
-Todos los mod posts deben abordar cuál es el problema, porque es un problema, y como arreglarlo. Revise el [Código de conducta para Modding y Mapping](/wiki/Rules/Code_of_conduct_for_modding_and_mapping#cómo-hacer-una-publicación-de-un-mod) para más detalles en cómo dar formato a los mod posts.
+Todos las publicaciones de los mods deben indicar cuál es el problema, por qué es un problema y cómo solucionarlo. Consulta el [código de conducta para el modding y el mapping](/wiki/Rules/Code_of_conduct_for_modding_and_mapping#cómo-hacer-una-publicación-de-un-mod) para obtener más información sobre el formato de las publicaciones de los mods.
```
Comment (agatemosu) [inline wiki/Modding/es.md:30]:
```suggestion
Todos las publicaciones de mods deben indicar cuál es el problema, por qué es un problema y cómo solucionarlo. Consulta el [código de conducta para el modding y el mapping](/wiki/Rules/Code_of_conduct_for_modding_and_mapping#cómo-hacer-una-publicación-de-un-mod) para obtener más información sobre el formato de las publicaciones de mods.
```

File: wiki/Modding/es.md [hunk 3]
```diff
@@ -2,41 +2,41 @@
...
 
-Los beatmappers regularmente ofrecen [kudosu](/wiki/Modding/Kudosu) a los mod posts que ellos consideren como útiles al cliquear el botón de votar *pulgar arriba*. Esta estadística es rastreada por medio del perfil del usuario y se toma en cuenta al aplicar para ingresar a los [Beatmap Nominators](/wiki/People/Beatmap_Nominators).
+Los mappers suelen dar [kudosu](/wiki/Modding/Kudosu) a los mods que consideren útiles haciendo clic en el icono de *pulgar hacia arriba*. Esta estadística se rastrea en el perfil del usuario y se usa como referencia al solicitar el ingreso a los [Beatmap Nominators](/wiki/People/Beatmap_Nominators).
 
-## Tipos de mod posts
+## Tipos de las publicaciones de mods
```
Comment (agatemosu) [inline wiki/Modding/es.md:34]:
```suggestion
## Tipos de publicaciones de mods
```

File: wiki/Modding/es.md [hunk 4]
```diff
@@ -2,41 +2,41 @@
...
 
-El objetivo de cada post de un `Problema` o una `Sugerencia` en una página de discusión de beatmap es explicar todo lo relacionado con un beatmap en el contexto de su diseño.
+El objetivo de cada publicación de un `Problema` o `Sugerencia` en la página de discusión de un beatmap es explicar todo lo relativo a un beatmap en el contexto de su diseño.
```
Comment (agatemosu) [inline wiki/Modding/es.md:36]:
```suggestion
El objetivo de cada publicación con un `Problema` o una `Sugerencia` en la página de discusión de un beatmap es explicar todo lo relacionado con un beatmap en el contexto de su diseño.
```

Comment (agatemosu) [reply wiki/Modding/es.md:36]: no sé dónde está eso de problemas y sugerencias

Comment (UnknownOsu) [reply wiki/Modding/es.md:36]: [Abajo a la derecha al entrar en la discusión de un mapa](https://i.imgur.com/N5dwT4i.png)

**Review by agatemosu — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**